server/controllers/refer/courseParticipationsController.ts

```diff
@@ -48,6 +48,29 @@ export default class CourseParticipationsController {
     }
   }
 
+  details(): TypedRequestHandler<Request, Response> {
```

Comment:
Are we ok for with `details` for the view of the page?  Just thinking ahead to the naming of the `PUT` method.  Would we want that to be `update`/`submit` or something else?

Reply:
I _was_ wondering about adding a new `courseParticipationDetailsController` with its own `show` and `update` method, but with the number of pages being added in this flow, eventually, would that be over the top?

Reply:
I can live with all of this journey being in one controller, if that's easier, as long as we have a consistent means of `updating` etc? Could have an `updateDetails`-style method, but I know it's slightly inconsistent with the rest of our service. However, this is the first time we've got these sub-pages, so it's a new thing.

Reply:
I agree, makes sense to follow the pattern of the other pages and have a new controller 👍